package.json

```diff
@@ -62,15 +62,15 @@
     "eslint": "^1.7.3",
     "express": "^4.14.0",
     "express-graphql": "^0.5.4",
-    "graphql": "^0.9.0",
+    "graphql": "^0.10.1",
     "graphql-relay": "^0.4.3",
     "isparta": "^4.0.0",
     "istanbul": "^0.4.0",
     "mocha": "^3.0.1",
     "mysql": "^2.11.1",
     "pg": "^5.0.0",
     "pg-hstore": "^2.3.2",
-    "sequelize": "^3.24.6",
+    "sequelize": "github:sequelize/sequelize#master",
```

Comment:
remove direct dependencies for this and dataloader-sequelize and i can merge, the changes to src/relay looks reasonable enough